fly.toml

```diff
@@ -0,0 +1,19 @@
+# fly.toml app configuration file generated for ever-teams on 2023-10-26T08:19:45Z
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "ever-teams"
+primary_region = "lax"
+kill_signal = "SIGINT"
+kill_timeout = "5m0s"
+
+[build]
```

Comment:
@evereq 

I think we should use the image link which is already built and pushed (GitHub or Docker Hub)
Just like you did in gauzy

https://github.com/ever-co/ever-gauzy/blob/e875f02e4e9f417252c3322a5cc8311d3372476f/.fly/applications/webapp/fly.toml#L5-L8

Reply:
@paradoxe35  yes, so we have 3 ways to make it work for Ever Teams:

1) one is to use Docker and run docker in Fly.io (and other platforms), similarly to how we do it in Ever Gauzy

2) maybe Fly.io support natively running NodeJs apps? If so, it's basically similar to Vercel.com, so we should use that instead of Docker for Fly.io, but we MAY need Docker anyway for other platforms.

3) we can build Ever Teams maybe as "static" ("export" from NextJs) and deploy into some CDN in Fly.io and others (e.g. to Storage like S3 with Cloudfront in front of it, of course, what is available in Fly.io etc). However, I am NOT sure at this stage Ever Teams will work like that, most probably we are using some SSR already or something like that, hard to tell. 
 
So yes, if we do it as 1) above, we probably have to push to some public registry Docker image and run it in Fly.io and other services. You can find relevant GitHub actions inside Gauzy project

Reply:
From what I have seen so far
I works well with Docker

Reply:
I'm going to create GitHub actions then @evereq

Reply:
Ok, I will add required secrets on Github, let me know list of env var names

Reply:
Sure @evereq